### DISCUSSION TITLE
Disable Alt+F binding

### DISCUSSION THREAD

**Discussion by wilkgr76** [vercel/hyper#6471] [Q&A] [answered]
I'm using `fish`, and Alt+F auto-completes the next argument. Unfortunately, in hyper, the Alt+F keybind opens the menu. 

I can see that the default keymaps includes this:

`"window:hamburgerMenu": "alt+f"`

but I'm not sure how to disable using my config file.

**Comment by LabhanshAgrawal** [comment, marked as answer]
you can set some other binding from the `keymaps` key in the config

**Reply by wilkgr76** [reply]
Is there any way to disable the binding completely?

**Reply by LabhanshAgrawal** [reply]
Maybe you can replace it with empty string
Or you may delete it from the keymaps file from the installation directory

**Reply by wilkgr76** [reply]
Replacing with empty string worked - thank you!

**Comment by persiklover** [comment]
I use wsl (Ubuntu 20.04) and I want to use `Ctrl+Shift+D` in vim.
Disabling hyper command like this `"pane:splitRight": ""` helps but vim seems to ignore this exact combination of keys. I tried it in the default windows terminal and it works perfectly fine so my vim config is not the problem.

What do I do?
